src/bitmap/iter.rs

```diff
@@ -131,6 +131,25 @@ impl IntoIterator for RoaringBitmap {
     }
 }
 
+impl From<u32> for RoaringBitmap {
+    fn from(value: u32) -> RoaringBitmap {
+        let mut rb = RoaringBitmap::new();
+        rb.insert(value);
+        rb
+    }
+}
+
+impl<const N: usize> From<[u32; N]> for RoaringBitmap {
+    fn from(mut arr: [u32; N]) -> Self {
+        if N == 0 {
+            return RoaringBitmap::new();
+        }
+
+        arr.sort();
+        RoaringBitmap::from_sorted_iter(arr.into_iter()).unwrap()
```

Comment:
I would prefer that you don't do more than what this function should do and don't sort here.

```suggestion
        RoaringBitmap::from_iter(arr.into_iter())
```